scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -267,8 +267,8 @@ trait Config {
    */
   def addReducerEstimator(clsName: String): Config =
     update(Config.ReducerEstimators) {
-      case None => Some(clsName) -> ()
-      case Some(lst) => Some(clsName + "," + lst) -> ()
+      case None => Some(clsName) -> Unit
```

Comment:
Ahh, this is what we want here. So, let's change it to:

```scala
(Some(s"$clsName,$lst"), ())
(Some(clsName), ())
```
and just not use the `->` enrichment.